site/blog/OIS_Sessions_Blog

```diff
@@ -0,0 +1,53 @@
+---
+title: StarlingX sessions from Open Infrastructure Summit
+author: jeff-gowan
+date: 2019-05-22
+category: news
+---
+Did you miss out on seeing some of the StarlingX sessions at Open Infrastructure Summit? If so, don’t worry about it because this blog has you covered. 
+
+StarlingX was a hot topic at OIS this year. There were more than 15 sessions that included StarlingX in some manner. The project even got some good coverage in the keynote from Intel’s Melissa Evers-Hood. When she displayed a logo chart of contributors she shared an anecdote about one of them. She said one of the contributors told her, “I was going to do a competitive analysis of StarlingX…vs. the other projects that provide a hardened edge cloud…then realized [there are no comparisons](https://www.youtube.com/watch?v=lRR3mJrg6t8&feature=youtu.be&t=343).” This is a great illustration that helps explain why StarlingX was so popular at the event. 
```

Comment:
I would leave out 'StarlingX was a hot topic at OIS this year.' and also 'This is a great illustration that helps explain why StarlingX was so popular at the event.' and let the sessions speak for themselves.

You can say that this was a great Summit for the community and the project got a lot of representation at the conference and this blog post will highlight some of the sessions that people can watch the recording of.

Reply:
Or something like this: "StarlingX made a strong showing at OIS this year, with more than 15 sessions that covered aspects of the project from general overview down to technical details and community building. It even garnered a mention in the keynote from Intel’s Melissa Evers-Hood..."